Description: powerpc32: fix 64-bit comparison
 On powerpc32 64-bit comparison code generated dangling
 target labels. This caused ghc build failure as:

    $ ./configure --target=powerpc-unknown-linux-gnu && make
    ...
    SCCs aren't in reverse dependent order
    bad blockId n3U

 This happened because condIntCode' in PPC codegen generated
 label name but did not place the label into `cmp_lo` code block.

 The change adds the `cmp_lo` label into the case of negative
 comparison.
 Signed-off-by: 's avatarSergei Trofimovich <slyfox@gentoo.org>
 .
Origin: https://gitlab.haskell.org/ghc/ghc/commit/25dce3fc05e4788240ac6d192919063a9f548f7f
Last-Update: 2019-08-09

--- ghc-8.6.5+dfsg1.orig/compiler/nativeGen/PPC/CodeGen.hs
+++ ghc-8.6.5+dfsg1/compiler/nativeGen/PPC/CodeGen.hs
@@ -923,6 +923,7 @@ condIntCode' True cond W64 x y
                  , BCC LE cmp_lo Nothing
                  , CMPL II32 x_lo (RIReg y_lo)
                  , BCC ALWAYS end_lbl Nothing
+		 , NEWBLOCK cmp_lo
                  , CMPL II32 y_lo (RIReg x_lo)
                  , BCC ALWAYS end_lbl Nothing
 
